ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2389,6 +2389,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-nutanix-ipi
+- as: vsphere-8-ipi-ovn-etcd-encryption-p2-f7
```

Comment:
Could you add upgrade tests, too ?

Reply:
@liangxia the vsphere-8 support is newly added in 4.13, I guess we do not need to update it upgrade tests, right?

Reply:
Ack